pkg/stub/handler.go

```diff
@@ -42,6 +43,7 @@ const (
 	regkey                 = "keyForSamplesRegistryField"
 	skippedstreamskey      = "keyForSkippedImageStreamsField"
 	skippedtempskey        = "keyForSkippedTemplatesField"
+	initialprocessingkey   = "keyForIndicatingInitialProcessing"
```

Comment:
any sort of "we need to know this is the first time because we do something special" logic makes me nervous/wary.

Reply:
per comment below removing (resolving my paranoia)

Reply:
yeah, paranoia is valid, but we should try to solve it by inspecting the current state of the system before adding new state (since that new state can in turn end up wrong/corrupt/invalid/racey)